File: FastScript/Demos/Main/Samples/BasicScript/string.vb
' FastScript v1.0          
' String demo              


dim i, j
dim s as String


  s = "Hello World!\r\nIt's working!"

  j = 0
  for i = 1 to Length(s)
    if s[i] = " " then Inc(j)
  next

  ShowMessage(s)
  ShowMessage("spaces: " & j)

    

File: FastScript/Demos/Main/Samples/JScript/string.js
//****************************/
//* FastScript v1.0          */
//* String demo              */
//****************************/


  var s = "Hello World!\r\nIt's working!";
  var j = 0;

  for(var i = 1; i <= Length(s); i++)
    if (s[i] == " ")
      j++;

  ShowMessage(j);

    